docs/maintainers/releasing.md

```diff
@@ -160,20 +160,14 @@ On Windows, the `-Runtime` parameter should be specified for `Start-PSBuild` to
 # Install dependencies
 Start-PSBootstrap -Package
 
-# Build for v6.0.0-beta.1 release targeting Windows 10 and Server 2016
-Start-PSBuild -Clean -CrossGen -PSModuleRestore -Runtime win10-x64 -Configuration Release -ReleaseTag v6.0.0-beta.1
+# Build for v6.0.0-beta.1 release targeting Windows
```

Comment:
>  On Windows, the `-Runtime` parameter should be specified for `Start-PSBuild` to indicate what version of OS the build is targeting.

This sentence needs to be updated to explain that `-Runtime` needs to be `win7-*` to build windows universal package.

> Build for v6.0.0-beta.1 release targeting Windows

Here we still should mention that `targeting Windows 7`.